.ci/Jenkinsfile

```
@@ -0,0 +1,92 @@
+#!groovy
+/**
+ * Jenkins pipeline to build Corda DJVM
+ */
+
+/**
+ * Kill already started job.
+ * Assume new commit takes precendence and results from previous
+ * unfinished builds are not required.
+ * This feature doesn't play well with disableConcurrentBuilds() option
+ */
+@Library('existing-build-control')
+import static com.r3.build.BuildControl.killAllExistingBuildsForJob
+killAllExistingBuildsForJob(env.JOB_NAME, env.BUILD_NUMBER.toInteger())
+
+/**
+ * Sense environment
+ */
+boolean isMaster = (env.BRANCH_NAME =~ /^master$/)
+
+pipeline {
+    agent {
+        label 'standard'
+    }
+
+    options {
+        timestamps()
+        buildDiscarder(logRotator(daysToKeepStr: '7', artifactDaysToKeepStr: '7'))
+        timeout(time: 3, unit: 'HOURS')
+    }
+
+    environment {
+        ARTIFACTORY_CREDENTIALS = credentials('artifactory-credentials')
+        ARTIFACTORY_BUILD_NAME = "DJVM / Publish / Publish to Artifactory".replaceAll("/", "::")
+    }
+
+    stages {
+
+        stage('Build DJVM') {
+            steps {
+                sh "./gradlew --daemon --no-build-cache clean build"
+            }
+        }
+
+        stage('Build DJVM Example') {
+            steps {
+                sh "cd djvm-example && ./gradlew --daemon --no-build-cache clean build && cd .."
```

please check first if I didn't mess up with the syntax
```suggestion
                dir('djvm-example') {
                    sh "./gradlew --daemon --no-build-cache clean build"
                }
```

---

Looks good